ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java

```diff
@@ -126,7 +126,7 @@ protected Optional<List<BlockHeader>> processResponse(
     }
 
     LOG.debug(
-        "Received {} of {} headers requested from peer {}",
+        "Received {} of {} headers requested from peer {}...",
         headersList.size(),
         count,
         peer.getShortNodeId());
```

Comment:
LOG.atDebug().setMessage(
        "Received {} of {} headers requested from peer {}...")
.addArgument(headersList::size)
.addArgument(count)
.addArgument(peer::getShortNodeId).log();